api/check_post.py

```diff
@@ -2,10 +2,10 @@
 
 from api.utils import get_payload
 
-post_api = Blueprint('post', __name__)
+check_post_api = Blueprint('post', __name__)
```

Comment:
bad variable name

Reply:
use `apod_api` or `picture_api` or `wallpaper_api` or something that indicates what endpoint is for (use part of your endpoint's name)